dnf5/commands/repoquery/repoquery.cpp

```diff
@@ -150,6 +260,22 @@ void RepoqueryCommand::load_additional_packages() {
     }
 }
 
+// In case of input being nevras -> resolve them to packages
+static libdnf::rpm::PackageSet resolve_nevras_to_packges(
+    libdnf::Base & base, const std::vector<std::string> & nevra_globs, const libdnf::rpm::PackageQuery & base_query) {
+    auto resolved_nevras_query = libdnf::rpm::PackageQuery(base, libdnf::sack::ExcludeFlags::APPLY_EXCLUDES, true);
```

Comment:
When I wrote use `PackageSet`, I really meant use. Not just change the return type of the function. As I wrote in the 3 line sample code. Replace this line with:
`auto resolved_nevras = libdnf::rpm::PackageSet(base);`

Reply:
Right, thank you again.